Dockerfile

```diff
@@ -8,5 +8,4 @@ COPY --from=builder /home/rust/src/target/x86_64-unknown-linux-musl/release/gtfs
 VOLUME /app/input
 VOLUME /app/output
 
-ENTRYPOINT ["./gtfs2ntfs"]
-CMD ["-i", "./input", "-o", "./output"]
+ENTRYPOINT ["./gtfs2ntfs", "-i", "./input", "-o", "./output"]
```

Comment:
Why changing this ? I think using a CMD for entrypoint options is a good practice

Reply:
this allow to simplify `docker run` and pass arguments that are not in `CMD`
before
```
docker run --rm -v path/to/input:/app/input -v path/to/output:/app/output gtfs2ntfs -i /app/input -o /app/output -p PREFIX
```

after
```
docker run --rm -v path/to/input:/app/input -v path/to/output:/app/output gtfs2ntfs  -p PREFIX
```